docs/2_registration_project_access_session.md

```diff
@@ -0,0 +1,142 @@
+> 
+> data/example_directory_1/sub_directory_1: example_file_1.txt
+> 
+> data/example_directory_2: example_file_2.txt      example_file_5.txt  
+> sub_directory_2
+> 
+> data/example_directory_2/sub_directory_2: example_file_3.txt     
+> example_file_4.txt
+
+
+
+The general upload command is [`dds data put`.](https://scilifelabdatacentre.github.io/dds_cli/data/#dds-data-put)
+~~~
+dds data put --project "<Project ID>" --source "<File or directory to upload>"
+~~~
+
+ - [ ] TASK: Upload the files in the data folder. Remember to change the **project ID** and the **source** from the command above
+
+If you have forgotten the project ID, you can list all active projects you have access with: [Docs](https://scilifelabdatacentre.github.io/dds_cli/project/#dds-project-ls)
+
+~~~
+dds project ls
+~~~
+
+# List contents
+
+You can interactively list the contents of a project with the [`ls` command:](https://scilifelabdatacentre.github.io/dds_cli/data/#dds-data-ls)
+~~~
+dds data ls --project "<Project ID>"
+~~~
+
+However, You can view all files and directories in a project as a tree structure, using the `--tree` option.
+
+~~~
+dds data ls --project "<Project ID>" --tree
+~~~
+![enter image description here](https://scilifelabdatacentre.github.io/dds_cli/_images/dds-data-ls-tree.svg)
+
+ - [ ] TASK: Verify that all the data has being uploaded sucesfully
+
+# Release project and download data
+
+
+1. In order for researchers to get access to the data, you need to [change the status](https://scilifelabdatacentre.github.io/dds_cli/project/#dds-project-access) to `Available`.
+
+~~~
+dds project status release --project "<Project ID>"
+~~~
+
+![enter image description here](https://scilifelabdatacentre.github.io/dds_cli/_images/dds-project-status-release.svg)
+##
+
+2. To [invite researchers](dds%20user%20add%20%5BEmail%20address%5D%20--role "&lt;Account role&gt;quot; --project quot;&lt;Project ID&gt;") that will download the data to the project:
+
+~~~
+dds project access grant --project "<Project ID>" --email "<email>"
+~~~
+
+or
+
+~~~
+dds user add [Email address] --role "Researcher" --project "<Project ID>"
+~~~
+
+##
+3. To download the full project contents
+~~~
+dds data get --get-all --project "<Project ID>" 
+~~~
+
+You can also use the `--source` option to specify which file or directory you want to download within the project. If you want to download multiple individual files or directories, specify the `--source` option multiple times.
+
+~~~
+dds data get --source "<1st file or directory>" --source "<2nd file or directory>" [... etc] --project "<Project ID>"
+~~~
+
+ - [ ] TASK: Release the project after making sure all the data was uploaded, remember to specify the correct project ID
+ - [ ] TASK: Download the project contents, and verify that they are the correct. You can download data with your unit admin account.
+ - [ ] Optional: Use another email you may have to invite yourself as a researcher, you don't need to register again.
```

Comment:
I get what you mean here but for some reason find this sentence hard to parse. Maybe could say:
_If you have another account on Test DDS, you can invite that email as a researcher_
That said, 1) couldn't they just use plus addressing to instantly invite themselves as a researcher anyway? 2) how does it work when you want to set up authentication for 2 different accounts on the same computer? Presumably they'd have to end/delete the stored session token and then re-auth with the other account's credentials. Might be good to mention that

Reply:
It is just so they could test adding emails as a researcher, I don't intend on them to log in again. Because that could be too much mess.

Reply:
I still don't really understand this one.

Reply:
Let me know if I'm misinterpreting etc.

But basically this is how it works:
- If they want to be able to download with a researcher account then they'd need to invite themselves as a researcher to that project, and then that researcher email would need to register an account. If that email has already been invited to the system and they have registered a researcher account previously, then they can invite that email to a project, and they then automatically have access to download, they wouldn't need to register again.
- In order to download with that researcher account, first they'd need to re-auth with the researcher credentials, and that would replace the existing local token (that belonged to the authenticated unit user account). 

So I think the "you don't need to register again" should be removed/moved somewhere else? If we want to have this optional suggestion here we could rephrase it to something like this, perhaps? (But we may want to include steps in that case)
> - [ ] Optional: If you have another email address, use your Unit Admin / Personnel account to invite that email as a researcher. Once you have registered the "researcher" account, authenticate using the researcher account credentials and use the `dds data get` command as described above to download the data.